fe/fe-core/src/test/java/com/starrocks/storagevolume/StorageVolumeTest.java

```diff
@@ -22,6 +22,7 @@
 import com.staros.proto.AzBlobFileStoreInfo;
 import com.staros.proto.FileStoreInfo;
 import com.staros.proto.FileStoreType;
+import com.staros.proto.HDFSFileStoreInfo;
```

Comment:
need add ut to cover the storage provider `strict mode` and default mode behavior. use UT to help explain the code design purpose.